src/cpp/desktop/DesktopUtils.cpp

```diff
@@ -45,6 +45,8 @@ using namespace rstudio::core;
 namespace rstudio {
 namespace desktop {
 
+class DesktopMainWindow;
```

Comment:
Should this be `MainWindow`? (looks like the classname below is just `MainWindow`)

Reply:
Oops, yes, nice catch. Already have the forward decl in the header, adding the actual include instead. Not strictly needed since non-Mac doesn't do anything with it yet, but just to be consistent.